EzGM/selection.py

```diff
@@ -281,6 +281,36 @@ def write(self, obj=0, recs=1, rtype='acc', recs_f=''):
         -------
         None.
         """
+        def rename_nga(filename, hist):
```

Comment:
Let's make the code more elegant. Nowadays, I try to make the codes as short as possible. I do not think we need these rename functions.